bluesky/pds_client.go

```diff
@@ -271,7 +215,7 @@ func (c *Client) Follow(
 
 // Unfollow removes any app.bsky.graph.follow for the subject from the account
 // the client is authenticated as.
-func (c *Client) Unfollow(
+func (c *PDSClient) Unfollow(
 	ctx context.Context, subjectDID string,
 ) error {
 	profile, err := c.GetProfile(ctx, subjectDID)
```

Comment:
TODO: Figure if calling GetProfile on /our/ PDS for the subject (who may be on a different PDS) is going to cause problems ??

Part of me thinks that AppView proxying may occur for this RPC and we should be fine - it would be a pain in the arse to have a PDSClient, BGSClient and AppViewClient. We should try calling GetProfile on one PDS, for a user on another PDS to check.